src/scrapeLinks.ts

```diff
@@ -27,8 +27,13 @@ const scrapeFromString: (filePath: string, content: string) => string[] = (
         /\[.*?\]\((?:<((?:\(.*?\)|.)*?)>|((?:\(.*?\)|.)*?))(?: ["'].*?["'])?\)/gm,
         (x) => x[2] || x[1]
       );
+      const mdRefLinks = regexMap(
+        content,
+        /\[.*?\]:\s*\n?\s*(.*)/gm,
```

Comment:
Had to simplify the inner part of this regex bc/ for some reason it wasn't matching URLs in this style.